docs/en/sql-reference/aggregate-functions/reference/initializeAggregation.md

```diff
@@ -0,0 +1,37 @@
+---
+toc_priority: 150
+---
+
+## initializeAggregation {#initializeaggregation}
+
+Initializes aggregation for your input rows. It is intended for the functions with the suffix `State`.
+Use it for tests or to process columns of types `AggregateFunction` and `AggregationgMergeTree`.
+
+**Syntax**
+
+``` sql
+initializeAggregation (aggregate_function, column_1, column_2);
+```
+
+**Parameters**
+
+-   `aggregate_function` — Тname of the aggregation function. The state of this function — the creating one. [String](../../../sql-reference/data-types/string.md#string).
+-   `column_n` — The column to translate it into the function as it`s argument. [String](../../../sql-reference/data-types/string.md#string).
```

Comment:
```suggestion
-   `column_n` — The column to translate it into the function as it's argument. [String](../../../sql-reference/data-types/string.md#string).
```

Reply:
Backtick `` ` `` is a markup symbol in Markdown. Please don't use it as a punctuation symbol. Otherwise, you can try to escape it using backslash `` \` ``.